SparkPost/matchbox
# UX-605 Set up Percy to collect snapshots

### What Changed
- Adds new test commands under `test:regression`
- Adds a new folder for Cypress tests only for visual regression testing under `cypress/regression`
- Adds a Github workflow to run visual regression tests on the `main` branch

### How To Test or Verify
- Run the tests locally, and verify Percy snapshots have been uploaded to https://percy.io/efff6832/matchbox


### PR Checklist
